chat-plugins/info.js

```diff
@@ -460,6 +288,15 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature." + "\n" +
+		"!data [pokemon/item/move/ability] - Show everyone these details. Requires: + % @ & ~",
+	detailshelp:
+		"/details [pokemon] - Get additional details on this pokemon/item/move/ability/nature." + "\n" +
+		"!details [pokemon] - Show everyone these details. Requires: + % @ & ~",
+	dthelp:
```

Comment:
I would generally support recursion rather than aliasing and manually reading `cmd`, as a way to implement multiple commands with one function. That would fix this issue, too.